kv/test_raftstore/cluster.go

```diff
@@ -48,15 +50,17 @@ func NewCluster(count int, schedulerClient *MockSchedulerClient, simulator Simul
 		snapPaths:       make(map[uint64]string),
 		simulator:       simulator,
 		cfg:             cfg,
+		baseDir:         "test-raftstore",
 	}
 }
 
 func (c *Cluster) Start() {
 	ctx := context.TODO()
 	clusterID := c.schedulerClient.GetClusterID(ctx)
 
+	c.CleanAbnormalTempFile()
```

Comment:
you should check whether the `TempDir` is not deleted after the test is finished instead of cleaning it afterwards

Reply:
but if test run successful, it will call `cluster.Shutdown` to remove all tmp; if test stop deliberately, the tmp cannot be deleted.

Reply:
You meaning that I should move this function to `test` rather than `cluster`?

Reply:
I find it is difficult to delete temp file if test fail due to some unexpected panic. So, at last, I changed it, and add auto delete in Makefile.